geomesa-accumulo/geomesa-accumulo-distributed-runtime/pom.xml

```diff
@@ -72,9 +74,6 @@
                             <transformers>
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
                             </transformers>
-                            <createDependencyReducedPom>false</createDependencyReducedPom>
-                            <shadedArtifactAttached>false</shadedArtifactAttached> <!-- replaces original artifact -->
-                            <finalName>geomesa-accumulo-distributed-runtime_${scala.binary.version}-${project.version}</finalName>
```

Comment:
Does this change affect the resultant output jar?

Reply:
no, it's the same - i tried to remove redundant configuration options that weren't needed.